packages/babel-plugin-react-jsx/__tests__/TransformJSXToReactJSX-test.js

```diff
@@ -29,15 +30,307 @@ function transform(input, options) {
           {
             useBuiltIns: true,
             useCreateElement: false,
-            ...options,
+            ...pluginOpts,
           },
         ],
       ],
+      ...babelOpts,
     }).code
   );
 }
 
 describe('transform react to jsx', () => {
+  it('throws error when sourceType is module and autoImport is require', () => {
+    let _error;
+    const code = `var x = <div><span /></div>`;
+    try {
+      transform(code, {
+        autoImport: 'require',
+      });
+    } catch (error) {
+      _error = error;
+    }
+    expect(_error).toEqual(
+      new SyntaxError(
+        'undefined: Babel `sourceType` must be set to `script` for autoImport ' +
```

Comment:
Why does the error start with 'undefined'?

Reply:
it looks like it's the filename if there is a `filename` associated with the code. `path.buildCodeFrameError` automatically adds this. Will remove from the test though